src/memorypad.cpp

```diff
@@ -298,6 +298,8 @@ struct MemoryPadTrackpad : OpaqueWidget {
     nvgCircle(args.vg, puckDot.x, puckDot.y, 8.0);
     nvgFillColor(args.vg, SCHEME_YELLOW);
     nvgFill(args.vg);
+
+    Widget::drawLayer(args, layer);
```

Comment:
Should we not be calling the direct superclass? (`OpaqueWidget::drawLayer(args, layer);`)

Reply:
``OpaqueWidget`` is derived from ``Widget``
https://vcvrack.com/docs-v2/structrack_1_1widget_1_1OpaqueWidget
https://vcvrack.com/manual/Migrate2#3-5-Draw-custom-light-like-widgets-at-full-brightness

Reply:
Perhaps this betrays my more limited C++ knowledge, but I thought one should call the direct super.